pkg/rpcclient/nns/contract.go

```diff
@@ -120,3 +160,235 @@ func itemsToRecords(arr []stackitem.Item) ([]RecordState, error) {
 	}
 	return res, nil
 }
+
+// Roots invokes `roots` method of contract.
+func (c *ContractReader) Roots() (uuid.UUID, result.Iterator, error) {
+	return unwrap.SessionIterator(c.invoker.Call(c.hash, "roots"))
```

Comment:
We need a special iterator type for it. Like `RecordIterator` but it should iterate over records (strings).